src/In.ProjectEKA.DefaultHip/Link/PatientRepository.cs

```diff
@@ -11,10 +11,12 @@ namespace In.ProjectEKA.DefaultHip.Link
     public class PatientRepository : IPatientRepository
     {
         private readonly string filePath;
+        private readonly string maskedFilePath;
```

Comment:
Codacy found an issue: [Remove this unread private field 'maskedFilePath' or refactor the code to use its value.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5811615)